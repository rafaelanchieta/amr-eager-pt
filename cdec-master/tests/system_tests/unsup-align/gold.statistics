-lm_nodes 2
-lm_edges 3
-lm_paths 2
+lm_nodes 2
+lm_edges 3
+lm_paths 2
+lm_trans blue
constr_nodes 2
constr_edges 2
constr_paths 1
-lm_nodes 2
-lm_edges 4
-lm_paths 3
+lm_nodes 2
+lm_edges 4
+lm_paths 3
+lm_trans house
constr_nodes 2
constr_edges 2
constr_paths 1
-lm_nodes 4
-lm_edges 16
-lm_paths 49
+lm_nodes 7
+lm_edges 20
+lm_paths 49
+lm_trans the house
constr_nodes 7
constr_edges 10
constr_paths 4
-lm_nodes 4
-lm_edges 12
-lm_paths 25
+lm_nodes 7
+lm_edges 16
+lm_paths 25
+lm_trans house blue
constr_nodes 7
constr_edges 10
constr_paths 4
-lm_nodes 4
-lm_edges 14
-lm_paths 36
+lm_nodes 7
+lm_edges 18
+lm_paths 36
+lm_trans the the
constr_nodes 7
constr_edges 10
constr_paths 4
-lm_nodes 2
-lm_edges 5
-lm_paths 4
+lm_nodes 2
+lm_edges 5
+lm_paths 4
+lm_trans the
constr_nodes 2
constr_edges 2
constr_paths 1
-lm_nodes 4
-lm_edges 14
-lm_paths 36
+lm_nodes 7
+lm_edges 18
+lm_paths 36
+lm_trans the the
constr_nodes 7
constr_edges 10
constr_paths 4
-lm_nodes 4
-lm_edges 10
-lm_paths 16
+lm_nodes 7
+lm_edges 14
+lm_paths 16
+lm_trans thet thet
constr_nodes 7
constr_edges 10
constr_paths 4
