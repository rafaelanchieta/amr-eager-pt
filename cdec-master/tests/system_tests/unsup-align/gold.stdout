bleue ||| blue ||| 0-0
maison ||| house ||| 0-0
la maison ||| the house ||| 0-0 1-1
maison bleue ||| blue house ||| 0-1 1-0
la fleur ||| the flower ||| 0-0 1-1
la ||| the ||| 0-0
la bouche ||| the mouth ||| 0-0 1-1
le fin ||| thet end ||| 0-0 1-1
